docs/tutorials/get_started.ipynb

```diff
@@ -0,0 +1,240 @@
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "# Create a new circuit with 2 qubits and 2 classical bits (cbits)\n",
+    "qc = QuantumCircuit(2, 2)\n",
+    "\n",
+    "# Add a Hadamard gate to qubit 0\n",
+    "qc.h(0)\n",
+    "\n",
+    "# Perform a controlled-X gate on qubit 1, controlled by qubit 0\n",
+    "qc.cx(0, 1)\n",
+    "\n",
+    "# Measure qubit 0 to cbit 0, and qubit 1 to cbit 1\n",
+    "qc.measure(0, 0)\n",
+    "qc.measure(1, 1)\n",
+    "\n",
+    "# Return a drawing of the circuit using MatPlotLib (\"mpl\").\n",
+    "# Remove the \"mpl\" argument to get a text drawing.\n",
+    "qc.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "0c957de9",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "See :class:`~.QuantumCircuit` in the documentation for all available operations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77878c23",
+   "metadata": {},
+   "source": [
+    "## Run your quantum program\n",
+    "\n",
+    "Quantum circuits can produce random results, so you'll often want to collect a sample of the outputs by running the circuit many times. The following code samples the circuit using the `Sampler` class. \n",
+    "\n",
+    "This tutorial uses Qiskit's built-in `Sampler`, which can only handle small numbers of qubits. You can replace this with samplers of high-performance simulators from [Qiskit Aer](https://github.com/Qiskit/qiskit-aer#readme), or with samplers that send your circuit over the internet to run on real quantum computers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "69a8d872",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SamplerResult(quasi_dists=[{0: 0.4999999999999999, 3: 0.4999999999999999}], metadata=[{}])"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.primitives import Sampler\n",
+    "\n",
+    "# Create a Sampler object\n",
+    "sampler = Sampler()\n",
+    "\n",
+    "# Submit the circuit to the sampler\n",
+    "job = sampler.run(qc)\n",
+    "\n",
+    "# Once the job is complete, get the result\n",
+    "job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc5ce1eb",
+   "metadata": {},
+   "source": [
+    "The `qasi_dists` property is a list of sampled distributions for each circuit we provided. In this case we only provided one circuit, so the list only has one element.\n",
+    "\n",
+    "The distribution is a dictionary-like object. The keys are the measured outputs, and the values are the fraction of the time that output was measured. In this case, `0` was measured ~50% of the time, and `3` (which is `11` in binary) was also measured ~50% of the time.\n",
+    "\n",
+    "And that's it! You now have everything you need to write and run quantum programs."
```

Comment:
this line feels very final but then there is still another section about plotting results so it feels a bit disjointed. Could you move this line to the end of the notebook, maybe before the qiskit community section?